examples/rabbit_fox_hawk.jl

```diff
@@ -151,7 +151,7 @@ function initialize_model(
     end
     for _ in 1:n_foxes
         pos = random_walkable(model, model.landfinder)
-        add_agent!(pos, Rabbit, model, v0, rand(abmrng(model), Δe_rabbit:2Δe_rabbit))
+        add_agent!(pos, Fox, model, v0, rand(abmrng(model), Δe_rabbit:2Δe_rabbit))
```

Comment:
I'm confused, we add a fox but we have a `Δe_rabbit`?

Reply:
I was confused to, but

`    Δe_rabbit = 30,  ## energy gained from eating one rabbit`

Reply:
It doesn't make much sense anyway though as an initialization

Reply:
or maybe yes, it's like they start with some rabbit already eaten :D